README.md

```diff
@@ -143,7 +147,7 @@ export CUSTOM_SEARCH_ENGINE_ID="YOUR_CUSTOM_SEARCH_ENGINE_ID"
 
 ## Redis Setup
 
-Install docker desktop.
+Install Docker Desktop.
```

Comment:
Many people aren't using Docker Desktop for OCI compliant containers these days.

I've tested this with Rancher Desktop and it works with the "containerd" engine and "nerdctl" as well. You simply have to change the docker-compose line to "nerdctl compose"